### PR TITLE
Sync with development branch

### PR DESCRIPTION
**Pull request by luisfelipefurlan**
* **Please check if the PR fulfills these requirements**
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
feature

* **What is the current behavior?** (You can also link to an open issue here)
The code is not up to date with the development branch

* **What is the new behavior (if this is a feature change)?**
The code has been updated with development changes

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
dojot/dojot#2110

* **Other information**:

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -30,5 +30,5 @@ node_modules/
 # Ignore generate configs
 config/*.json
 
-#local history
-.history
```
Comment (mprevide) [inline .gitignore:34]: Add .history

Comment (luisfelipefurlan) [reply .gitignore:34]: .history was added twice

Comment (mprevide) [reply .gitignore:34]: Ok, sorry

**Comment by mprevide** [comment]
Do you really need to change the package-lock? I don't think so, since you didn't change anything in the package...
Revert the changes and always try to use the correct node version, in case it is the q is in the dockerfile.

**Review by mprevide — APPROVED** [review]

**Comment by mprevide** [comment]
please, fix the link with the issue change to dojot/dojot#issue

**Pull request merged**